test/integration/aab_offline_test.go

```diff
@@ -37,6 +37,9 @@ func TestOffline(t *testing.T) {
 				if runtime != "docker" && NoneDriver() {
 					t.Skipf("skipping %s - incompatible with none driver", t.Name())
 				}
+				if runtime == "crio" && runningDockerDriver(StartArgs()) {
```

Comment:
this line will be not needed next week after adding crio to the dockerfile. 
if want to keep this, please add a comment with issue link that this will need to be removed once that is fixed.

Reply:
gotcha, removed